internal/grpc/server/employee.go

```diff
@@ -0,0 +1,97 @@
+package server
+
+import (
+	"context"
+	"errors"
+	"github.com/dilyara4949/employees-api/internal/domain"
+	pb "github.com/dilyara4949/employees-api/proto"
+)
+
+type EmployeeServer struct {
+	Repo domain.EmployeesRepository
+	pb.UnimplementedEmployeeServiceServer
+}
+
+func (s *EmployeeServer) GetAll(empty *pb.Empty, stream pb.EmployeeService_GetAllServer) error {
+	employees := s.Repo.GetAll(context.Background())
+	for _, emp := range employees {
+		if err := stream.Send(employeeToProto(&emp)); err != nil {
+			return err
```

Comment:
Also, yeak, I kind of missed that, but errors for grps do not return like that. Please refer to the documentation about `status.Status` and `status.Error`

Reply:
fixed